### DISCUSSION TITLE
When max_file_size exceeded the temporary files in /tmp are not cleaned up

### DISCUSSION THREAD

**Discussion by core45** [spatie/laravel-medialibrary#3421] [Ideas]
When the size of the processed file is greater that set in max_file_size the process dies and the temporary files in /tmp are not cleaned up and it eventually leads to fill up all the available disk space and server lock.

spatie/laravel-medialibrary - version 10.12.2
Laravel v 10.10
PHP 8.2

**Comment by AlejandroAkbal** [comment]
Happening to me too

**Comment by freekmurze** [comment]
I'd accept a PR that fixes this.
